website/common/script/ops/buy/purchase.js

```diff
@@ -144,8 +144,11 @@ module.exports = function purchase (user, req = {}, analytics) {
     throw new NotAuthorized(i18n.t('notEnoughGems', req.language));
   }
 
-  let itemInfo = getItemInfo(user, type, item);
-  removeItemByPath(user, itemInfo.path);
+  let singlePurchaseTypes = ['gear'];
```

Comment:
could you move this types-array and `acceptedTypes` outside of the method? it doesn't need to be initialized on every method call since its static anyway